docs/sql_reference/src/asciidoc/_chapters/olap_functions.adoc

```diff
@@ -133,12 +145,21 @@ GROUP BY num, workgroupnum, annualsalary;
 [[limitations_for_window_functions]]
 == Limitations for Window Functions
 
-These limitations apply to all window functions.
+Trafodion supports many ANSI compliant OLAP window functions. These functions include AVG, COUNT, DENSE_RANK, MAX, MIN, RANK, ROW_NUMBER, STDDEV, SUM, and VARIANCE. However, Trafodion imposes certain limitations on all window functions.

```

Comment:
This is correct, since it says "include". But since you are already editing, would you want to include LEAD, LAG, FIRST_VALUE, LAST_VALUE?